packages/flamegrill/src/flamegrill.ts

```diff
@@ -31,15 +31,28 @@ export interface CookResults {
   [scenarioName: string]: CookResult;
 }
 
+
+/**
+ * Function that takes a directory path
+ * and creates the structure if it doesn't exist
+ * @param {string} path directory path
+ * @returns {string} the directory path
+ */
+function resolveDir(dirPath: string): string {
+  fs.mkdirSync(dirPath);
```

Comment:
I think you'll need an `fs.existsSync` check too. CI test seems to be failing on directory already existing.